pkg/kubelet/pod_workers.go

```diff
@@ -215,6 +215,8 @@ type PodWorkers interface {
 	// deleting a terminating static pod from the apiserver before the pod is shut
 	// down.
 	IsPodForMirrorPodTerminatingByFullName(podFullname string) bool
+
+	ForgerWorker(uid types.UID)
```

Comment:
what is a ForgerWorker?

Reply:
This function is used to clear the currently working pod (blocked pod), because no subsequent messages can be received in the blocking. After cleaning, forced deletion can work

Reply:
nit: This sounds to me like a helper method, an implementation detail, not something which should be exposed in PodWorkers interface. You might also consider adding an extra test case for this scenario?

Reply:
I think this method belongs to the PodWorkers interface, because its purpose is to provide a new channel for the force stop, and to reject the update before the force stop. Because when the force stop is initiated, all other updates and stops become unimportant. This method ensures the correctness of the force stop.
The use case for this scenario is actually relatively simple, the logic is that after the first stop, if the gracePeriod time has not yet arrived, it will not be possible to initiate a force stop. forgerWorker can make the force stop executed immediately.